tests/fafnir_tests/mod.rs

```diff
@@ -145,4 +179,13 @@ pub fn main_test(es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapper)
         .find(|&p| p.key == "amenity")
         .unwrap();
     assert_eq!(amenity_tag.value, "cafe".to_string());
+
+    // Test Address
+    let address_ocean_poi = &ocean_poi.address.as_ref().unwrap();
+    let label = get_label(address_ocean_poi);
```

Comment:
same here put the `&` at the `get_label` call

Reply:
done.